docs/intro/tutorial.rst

```diff
@@ -490,30 +523,109 @@ Another common pattern is to build an item with data from more than one page,
+
+Here is another spider that illustrates callbacks and following links,
+this time for scraping author information::
+
+
+    import scrapy
+
+
+    class AuthorSpider(scrapy.Spider):
+        name = 'author'
+
+        start_urls = ['http://quotes.toscrape.com/']
+
+        def parse(self, response):
+            # follow links to author pages
+            for href in response.css('.author a::attr("href")').extract():
+                yield scrapy.Request(response.urljoin(href),
+                                     callback=self.parse_author)
+
+            # follow pagination links
+            next_page = response.css('li.next a::attr("href")').extract_first()
+            if next_page is not None:
+                next_page = response.urljoin(next_page)
+                yield scrapy.Request(next_page, callback=self.parse)
+
+        def parse_author(self, response):
+            def extract_with_css(query):
+                return response.css(query).extract_first().strip()
+
+            yield {
+                'name': extract_with_css('h3.author-title::text'),
+                'birthdate': extract_with_css('.author-born-date::text'),
+                'bio': extract_with_css('.author-description::text'),
+            }
+
+This spider will start from the main page, it will follow all the links to the
+authors pages calling the ``parse_author`` callback for each of them, and also
+the pagination links with the ``parse`` callback as we saw before.
+
+The ``parse_author`` callback defines a helper function to extract and cleanup the
+data from a CSS query and yields the Python dict with the author data.
+
+Another interesting thing this spider demonstrates is that, even if there are
+many quotes from the same author, we don't need to worry about visiting the
+same author page multiple times. By default, Scrapy filters out duplicated
+requests to URLs already visited, avoiding the problem of hitting servers too
+much because of a programming mistake. This can be configured by the setting
+:setting:`DUPEFILTER_CLASS`.
 
 .. note::
-    As an example spider that leverages this mechanism, check out the
-    :class:`~scrapy.spiders.CrawlSpider` class for a generic spider
-    that implements a small rules engine that you can use to write your
+    As another example spider that leverages the mechanism of following links,
+    check out the :class:`~scrapy.spiders.CrawlSpider` class for a generic
+    spider that implements a small rules engine that you can use to write your
     crawlers on top of it.
 
-Storing the scraped data
-========================
+Using spider arguments
+======================
 
-The simplest way to store the scraped data is by using :ref:`Feed exports
-<topics-feed-exports>`, with the following command::
+You can provide command line arguments to your spiders by using the ``-a``
+option when running them::
 
-    scrapy crawl quotes -o items.json
+    scrapy crawl quotes -o items.json -a tag=humor
```

Comment:
If users will follow the tutorial as-is, items.json file will contain multiple json objects because by default data is appended. I think we should mention this gotcha somwhere.